src/main/scala/mesosphere/marathon/core/pod/Volume.scala

```diff
@@ -0,0 +1,17 @@
+package mesosphere.marathon.core.pod
+
+sealed trait Volume extends Product with Serializable {
```

Comment:
force subclasses to be case classes. i think we've adopted this pattern elsewhere as well